tailwind.config.js

```diff
@@ -29,6 +29,7 @@ module.exports = {
       events: "url('/assets/backgrounds/events.jpg')",
       contact: "url('/assets/backgrounds/contact.jpg')",
       aksaraBox: "url('/assets/backgrounds/aksaraBox.png')",
+      bernard: "url('/assets/backgrounds/bernard.jpg')",
```

Comment:
Is there any reason why you put the link inside the tailwind config and not putting the url directly

Reply:
Hem since the rest of the pages do that i guess for consistency sake?

Reply:
lol I think its fine its just that if everyone keeps doing this, the config becomes very dirty